data/projects.js

```diff
@@ -147,5 +147,119 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Web", "Blog"],
 		"markdown": "coala_web.md"
+	},
+	{
+		"name" : "Optimize Caching",
+		"desc" : "Improve coala's performance by implementing caching strategies.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs, \
+			especially <https://github.com/coala/cEPs/blob/master/cEP-0002.md>",
```

Comment:
Yes the requirement section can render markdown now, so we gotta use 
```
especially [cEP 0002](http://github.com...)
```
to generate links